package.json

```diff
@@ -2,6 +2,7 @@
   "name": "bunyan-category",
   "version": "0.1.3",
   "description": "A wrapper around Bunyan that provides support for log categorization and category-based configuration",
+  "repository": "github:JupiterOne/node-bunyan-category",
```

Comment:
This format is supported per the docs: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository